tc/aten/aten_compiler.h

```diff
@@ -88,6 +88,15 @@ void uncheckedRun(
     const Executor& executor,
     const std::vector<at::Tensor>& inputs,
     std::vector<at::Tensor>& outputs);
+
+/// This is the "low-latency" mode in which we just propagate raw pointers
+/// Sizes are not checked and it is the user's responsibility to ensure that
+/// they match. If the user doesn't then segfault will likely occur.
+template <typename Executor>
+void uncheckedRun(
+    const Executor& executor,
+    const std::vector<const void*>& inputs,
+    std::vector<void*>& outputs);
```

Comment:
The definition above has this argument as _const_ reference.  I'm surprised the compiler did not complain, maybe it never gets instantiated.